metadata.json

```diff
@@ -49,7 +49,7 @@
   ],
   "dependencies": [
     {"name":"puppetlabs/stdlib","version_requirement":">=3.0.0 <5.0.0"},
-    {"name":"puppetlabs/apt","version_requirement":">=1.8.0 <2.0.0"},
+    {"name":"puppetlabs/apt","version_requirement":">=1.8.0 <2.0.2"},
```

Comment:
For this case, the version required is 2.0.0 to 2.0.2 (since the apt module syntax changed from 1.8.0)